api/app/Models/Pool.php

```diff
@@ -41,6 +44,7 @@ class Pool extends Model
         'description' => 'array',
         'operational_requirements' => 'array',
         'key_tasks' => 'array',
+        'expiry_date' => 'date',
```

Comment:
I think we'll need new fields for the "Your Impact", "Essential Skills", "Asset skills", and "Other Requirements" sections, right?